client/src/components/ReviewSummary/ReviewSummary.module.css

```diff
@@ -1,12 +1,34 @@
 .block {
   display: block;
-  border: 0.5px solid grey;
 }
 
 .inline_block {
   display: inline-block;
-  width: 50%;
-  white-space: normal;
+  width: 49%;
+  padding: 1px;
+  word-wrap: break-word!important;
```

Comment:
Try to avoid using !important unnecessarily.

"When an important rule is used on a style declaration, this declaration overrides any other declarations. Although technically !important has nothing to do with specificity, it interacts directly with it. Using !important, however, is bad practice and should be avoided because it makes debugging more difficult by breaking the natural cascading in your stylesheets. When two conflicting declarations with the !important rule are applied to the same element, the declaration with a greater specificity will be applied."